packages/components/Input/src/Input.types.ts

```diff
@@ -127,11 +127,33 @@ export interface InputTokens extends InputCoreTokens {
 }
 
 export interface InputProps extends PressableFocusProps {
+  /*
+   ** An accessibility label for screen readers. Set on the text input.
+   */
+  accessibilityLabel?: string;
+
+  /*
+   ** An accessibility label for screen readers. Set on the accessory icon pressable.
+   */
+  accessoryIconAccessibilityLabel?: string;
+
   /*
    * Source URL or name of the icon to show on the input.
+   *
+   * Based on fluent guidelines this icon should be an "outline" icon.
+   * FocusedStateIcon is applied onFocus and should be a "filled" icon.
    */
   icon?: IconSourcesType;
```

Comment:
rename to `defaultIcon `to keep it inline with the comment below.

Reply:
On similar - is icon an optional property?

Reply:
Alright renaming.
Yes, the icon is optional!